helmfile.d/helmfile-60.teams.yaml

```diff
@@ -187,7 +186,29 @@ releases:
           - azure{{ end }}
           {{- if $team | get "stack.sitespeed" false }}
           - sitespeed{{ end }}
-
   {{- end }}
   {{- end }}
+  {{- if hasKey $team "jobs" }}
+  {{- range $job := $team.jobs }}
+  - name: job-{{ $teamId }}-{{ $job.name }}
+    installed: {{ has $v.cluster.id $team.clusters }}
```

Comment:
the installed flag will come from the spec (users can temporarily disable them), so this is not correct